### PR TITLE
Longer timeout to remove silent nodes

### PR DESCRIPTION
**Pull request by howard-stearns**
https://highfidelity.atlassian.net/browse/BUGZ-134

### CODE REVIEW

File: libraries/shared/src/DependencyManager.h [hunk 1]
```diff
@@ -89,7 +89,7 @@ class DependencyManager {
     QHash<size_t, QSharedPointer<Dependency>> _instanceHash;
     QHash<size_t, size_t> _inheritanceHash;
 
-    mutable QMutex _instanceHashMutex;
+    mutable QMutex _instanceHashMutex { QMutex::Recursive };
```
Comment (howard-stearns) [inline libraries/shared/src/DependencyManager.h]: This is from a separate PR by Samuel, that is needed to make a working build, but which hasn't been merged yet due to jenkins issues.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19407/HighFidelity-Beta-PR15552-aeff5d9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19407/HighFidelity-Beta-Interface-PR15552-aeff5d9.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19407/HighFidelity-Beta-PR15552-aeff5d9.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19407/HighFidelity-Quest-Beta-PR15552-aeff5d9.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19407/HighFidelity-Beta-PR15552-aeff5d9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19407/HighFidelity-Beta-Interface-PR15552-aeff5d9.exe

**Pull request merged**